### PR TITLE
9.0: multiple rounding issues

### PR DESCRIPTION
**Pull request by nim-odoo**

### CODE REVIEW

**Comment by nim-odoo** [comment]
@wtaferner

File: addons/account/models/account.py [hunk 1]
```diff
@@ -669,7 +669,18 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
...
+        # precision of the currency.
+        # The context key 'round' allows to force the standard behavior.
+        round_tax = False if company_id.tax_calculation_rounding_method == 'round_globally' else True
+        round_total = True
+        if 'round' in self.env.context:
+            round_tax = bool(self.env.context['round'])
+            round_total = bool(self.env.context['round'])
+
+        if not round_tax:
             prec += 5
```
Comment (wtaferner) [inline addons/account/models/account.py]: Can you consider/discuss this thought of mine to be added here based on the issue/PR https://github.com/odoo/odoo/pull/8427?
```python
else:
    prec += 1
```

Comment (nim-odoo) [inline addons/account/models/account.py]: I prefer to stick to the original scope of this PR, and leave the discussion to the mentioned issue. What you suggest would introduce a change in behavior, therefore it needs to be considered with with care.

Comment (wtaferner) [inline addons/account/models/account.py]: Ok, I understand and agree (about the scope), but then you might eventually poke the persons personally who are in charge of this as the issue seems to be in sleep mode again and it is discussed with heat again and again without useful solution until now whereas I think my concept or something similar would resolve it once and for all ;-)

**Pull request merged**